### DISCUSSION TITLE
Transmission with NordVPN, stuck on api calls

### DISCUSSION THREAD

**Discussion by NoooX** [haugene/docker-transmission-openvpn#2781] [Q&A] [answered]
Hi,
My transmission docker container failed to start after about 1.5 years of running in my synology NAS, so I updated to the latest image and can't get it to work now after hours and hours of debugging.

So, I'm running on a Synology NAS with DSM 7.2.
Settings: 
![image](https://github.com/haugene/docker-transmission-openvpn/assets/18527610/f7f03c3e-88c2-4b89-8663-4cd57abf3622)

I can't get pass the curl -s https://api.nordvpn.com/v1/servers/countries most of the time. It is just frozen and will later stop the container.
Some times I have been lucky and manage to get pass the first api request but will be stuck on the next one, example the groups or technologies request.
If I ssh into the NAS and then "docker exec -it haugene-transmission-openvpn bash" I can ping google.com, so I do infact have a internet connection. But I can not do the curl command and get the data back. It is just frozen.

Logs: 
![image](https://github.com/haugene/docker-transmission-openvpn/assets/18527610/ac66b9e2-df4d-4370-a123-38c143394464)

Just for fun I did install an Ubuntu distro on WSL with docker and downloaded the latest image of transmission. 
That worked like a charm on the first try.

Anyone have any idea how to solve this or debug further?

**Comment by ilike2burnthing** [comment]
As you're on Synology, try https://github.com/haugene/docker-transmission-openvpn/issues/2429.

**Reply by NoooX** [reply]
I have already tried to start it with high privilege and without without success. Don't think it is a privilege failure I'm facing since it does not crash, just timeout after the curl command.

**Reply by ilike2burnthing** [reply]
Try specifying the country, category, and protocol, see if that makes any difference - https://haugene.github.io/docker-transmission-openvpn/provider-specific/

**Reply by NoooX** [reply, marked as answer]
Thanks for your reply.
I went for a break and then googled some more and found this reddit thread: https://www.reddit.com/r/synology/comments/fre8rn/how_to_configure_synology_firewall_to_allow/

Then it hit me that I configured the firewall in order to stop all DSM and FTP login attempts from China and India.
If I disabled the firewall it did work again! 😄 
I just added the IP 172.17.0.1-172.17.0.255 in the firewall rules. 

No I just get that the container is unhealthy and that it can not connect to the tracker. But that is an other story how I will try to solve that.
